smarts/core/agent_interface.py

```diff
@@ -172,6 +172,12 @@ class AgentsAliveDoneCriteria:
     """
 
 
+@dataclass(frozen=True)
+class EventConfiguration:
+    not_moving: int = 60
+    """How long to check the agent is not moving"""
+
```

Comment:
Currently `not_moving` is the only event I really see that has a configurable aspect to it

Reply:
`not_moving` also has the 1 meter value that could potentially be configurable.

I also wonder if `wrong_way` could be time configured as well to implicitly account for overtaking. But that is not for this PR.